src/utils/pytorch_onnx_inference.py

```diff
@@ -0,0 +1,262 @@
+import glob
```

Comment:
can we merge this with `predict_model.py`? Maybe adding an option for whether to do inference with pytorch or both pytorch and onnx?

Reply:
Yeah, for this script I can merge it to predict_model.py